ansible/vagrant/vagrant-ansible.yml

```diff
@@ -1,19 +1,20 @@
 - hosts: all
   sudo: yes
+  vars:
+    - public_iface: eth0
```

Comment:
Sorry! I should have said something. No way. Most modern computers don't even have an eth0.  Does ansible have any facts we could use? We've got to find something other than eth0.